silkrpc/txpool/transaction_pool.hpp

```diff
@@ -42,9 +43,26 @@
 namespace silkrpc::txpool {
 
 struct StatusInfo {
-   unsigned int queued;
-   unsigned int pending;
-   unsigned int base_fee;
+    unsigned int queued;
+    unsigned int pending;
+    unsigned int base_fee;
+};
```

Comment:
I would suggest some renaming here to improve readability:

- `queued` -> `queued_count`
- `pending` -> `pending_count`
- `base_fee` -> `base_fee_count`

Reply:
ok